controllers/os-suse-jeos/charts/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gardener-extension-os-{{- .Values.OperatingSystem.Name }} 
```

Comment:
> I would not make the name configurable here - otherwise I could name it `ubuntu` although it's only doing `jeos`.
> 
> replace all `{{- .Values.OperatingSystem.Name }} ` with `suse-jeos` and remove it from the `values.yaml`

The rationale was to allow easy copy/paste of the templates from os-common, without needing manual editing. I don't see it makes any harm.

Reply:
The problem is that the user of this Helm chart can overwrite this during deployment, and then we can get into the situation (name is `ubuntu` but actual controller only operates on `jeos`).

Reply:
Changed.